client/src/cli/queryParams.ts

```diff
@@ -25,25 +26,25 @@ export const queryParams = async (
     throw new Error("Please provide a source chain ID (--sourceChainId <id>)");
   }
   let defaultPath = path.resolve(path.join("app", "axiom"));
-  let inputFile = path.join(defaultPath, "data", "output.json");
-  if (options.input !== undefined) {
-      inputFile = options.input;
+  let inputsFile = path.join(defaultPath, "data", "proven.json");
+  if (options.inputs !== undefined) {
+      inputsFile = options.inputs;
   }
-  const outputJson = readJsonFromFile(inputFile);
+  const outputsJson = readJsonFromFile(inputsFile);
```

Comment:
This line is pretty confusing.  I think the changes:

- `outputsJson` -> `provenJson`
- `inputsFile` -> `provenFile`

could clarify things.  What do you think?